Tutorials/Creating-and-distributing-a-package/index.md

```diff
@@ -0,0 +1,500 @@
+
+If you want to make changes and push a new version you can do these steps:
+
+- Create the new package in the backoffice
+- Sync your code to Github
+- Go to Our and upload a new zip version
+- Set that to the current version
+- Optionally archive the previous one
+
+However, while working this way is definitely possible, and will work for everyone it is pretty time consuming and requires you to do a lot of things in different places. So let's make it a bit easier!
+
+Instead of going to the Backoffice and creating or updating your package from the package section each time, you can use the UmbPack tool to make smaller changes.
+
+:::note 
+Any changes to Umbraco content and schema is a lot easier to do from the backoffice, but if it is file-based UmbPack is quicker.
+:::
+
+If you have a look back in your your solution you will notice there is an almost empty package.xml file in the root:
+
+![Package.xml file in root][packagexml]
+
+If you open this package.xml file you will notice it has some default values in it, the only new part is under files:
+
+```xml
+  <files>
+    <folder path="src/Package.Workshop/App_Plugins/Package.Workshop" orgPath="App_Plugins/Package.Workshop" />
+    <file path="src/Package.Workshop/bin/release/Package.Workshop.dll" orgPath="bin/Package.Workshop.dll" />
+  </files>
+```
+
+Here you may notice that compared to your backoffice created package.xml it has not only a file elements, but also a folder element. The folder element is not part of the regular package schema, and something we've added in UmbPack which works with the `umbpack pack` command where it runs through the folder and adds each file within it in the final zip.
+
+So the xml above will turn into this which is compatible with the Umbraco package installer, and is exactly like what you got in hen creating the package from the backoffice:
+
+```xml
+<files>
+  <file>
+    <guid>Package.Workshop.dll</guid>
+    <orgPath>/bin</orgPath>
+    <orgName>Package.Workshop.dll</orgName>
+  </file>
+  <file>
+    <guid>dashboard.html</guid>
+    <orgPath>/App_Plugins/Package.Workshop/Dashboard</orgPath>
+    <orgName>dashboard.html</orgName>
+  </file>
+  <file>
+    <guid>dashboardController.js</guid>
+    <orgPath>/App_Plugins/Package.Workshop/Dashboard</orgPath>
+    <orgName>dashboardController.js</orgName>
+  </file>
+  <file>
+    <guid>dashboardService.js</guid>
+    <orgPath>/App_Plugins/Package.Workshop/Dashboard</orgPath>
+    <orgName>dashboardService.js</orgName>
+  </file>
+  <file>
+    <guid>package.manifest</guid>
+    <orgPath>/App_Plugins/Package.Workshop/Dashboard</orgPath>
+    <orgName>package.manifest</orgName>
+  </file>
+  <file>
+    <guid>en.xml</guid>
+    <orgPath>/App_Plugins/Package.Workshop/Dashboard/Lang</orgPath>
+    <orgName>en.xml</orgName>
+  </file>
+</files>
+```
+
+Before we try it out using UmbPack, try to run this command in the root of your site:
+
+```umbpack pack -h```
```

Comment:
```suggestion
```
umbpack pack -h
```
```

(I'm not sure why, but this breaks the layout for me, guess is backticks on one line?)

Reply:
Not sure what you are suggesting here either as it breaks.. Should I try to remove the backticks on their own lines?

Reply:
yeah,  I think so, maybe it would be ok when published, but everything after this in the resto of the text to review seemed part of the 'code' example?